tensorflow/lite/arena_planner.cc

```diff
@@ -237,9 +237,14 @@ TfLiteStatus ArenaPlanner::CalculateAllocations(int first_node, int last_node) {
     }
   }
 
+  // For the case if the graph is empty the node index can be negative since we
+  // substract from the active node, so the node_index can be zero for those
+  // cases
+  if ((active_node - 1) > 0) {
```

Comment:
Shouldn't this be: active_node > 0? Your condition fails for the active_node == 1 case.

Reply:
@jdduke , yes you are right, sorry for the miss, i have updated the code as per your suggestion.
Kindly check.

Regards
Amit